src/controllers/trip.controller.js

```diff
@@ -0,0 +1,35 @@
+import TripService from '../services/trip.service';
+import AuthenticationService from '../services/authentication.service';
+import responseHandlers from '../utils/responseHandlers';
+import statusCodes from '../utils/statusCodes';
+import customMessages from '../utils/customMessages';
+import utils from '../utils/authentication.utils';
+
+const { updatedResponse, errorResponse } = responseHandlers;
+const { decodeToken } = utils;
```

Comment:
could you remove this line, because you are not using `decodeToken` in this controller?

Reply:
It's no longer there